### ISSUE TITLE
`do_rescan` does not detect orchard notes

### ISSUE THREAD

**Issue by jarys** [zingolabs/zingolib#256]
Orchard notes are not detected after `do_rescan`. Sapling notes are detected without problem.
tldr test:
```rust
fn test_orchard_rescan() {
    // send some fund to orchard address owned by wallet
    ...
    // check the orchard balance
    let balance = lightclient.do_balance().await;
    assert_ne!(balance["orchard_balance"], 0);

    // now rescan and check the orchard balance again
    lightclient.do_rescan().await.unwrap();
    let balance = lightclient.do_balance().await;
    assert_ne!(balance["orchard_balance"], 0);  // fails :/
}
```
full test:
https://github.com/zingolabs/zingolib/blob/af51858cca7153eed05e0a9c147336d1497d5052/zingolib/src/lightclient/tests.rs#L1952-L2043

**Comment by AloeareV** [comment]
I'm sorry I didn't catch this earlier, the test framework in `zingolib/src/lightclient/tests.rs` doesn't support orchard properly. We've been in the process of phasing it out, all the new tests have been going in `zingocli/tests/integration_tests.rs`, so we can use an actual regtest instance instead of trying to make the fake transaction framework accommodate orchard.

**Comment by AloeareV** [comment]
Closed in favor of #261
